vite_ruby/lib/vite_ruby/config.rb

```diff
@@ -145,12 +148,13 @@ def resolve_config(**attrs)
     }
 
     # Internal: Default values for a Ruby application.
+    # Not able to load env via config/vite.rb or override via ViteRuby.configure, need to check to avoid this change
     def config_defaults(asset_host: nil, mode: ENV.fetch('RACK_ENV', 'development'), root: Dir.pwd)
       {
         'asset_host' => option_from_env('asset_host') || asset_host,
         'config_path' => option_from_env('config_path') || DEFAULT_CONFIG.fetch('config_path'),
         'mode' => option_from_env('mode') || mode,
-        'root' => option_from_env('root') || root,
+        'root' => option_from_env('root') || Dir.pwd + '/frontend-react',
```

Comment:
Not possible for us to pass as a configuration or some input, to skip hard coding here ?